documentation/cadence.md

```diff
@@ -0,0 +1,7 @@
+
+
+* Trello [board](https://trello.com/b/8TEQ0gH0) is used to track WIP
+
+* Daily standup is at 15:00 Pacific/ 22:00 UTC in [Zoom](https://zoom.us/j/471016936?pwd=cXh2RWFib2F4RU1ldVZPM3BNdkcrQT09)
+
+* Standup items can be added to the Trell Agenda [card](https://trello.com/c/xOgDRhFu/22-daily-standup-agenda) 
```

Comment:
> * Standup items can be added to the ~Trell~ Trello Agenda [card](https://trello.com/c/xOgDRhFu/22-daily-standup-agenda)

* Standup items can be added to the Trello Agenda [card](https://trello.com/c/xOgDRhFu/22-daily-standup-agenda)